modules/ROOT/pages/migration/index.adoc

```diff
@@ -16,14 +16,389 @@ To update your Neo4j GraphQL Library, use npm or the package manager of choice:
+
+Pagination and sorting also changed in version 6.
+
+*Top-level field pagination and sorting*
+
+Version 5:
+```graphql
+query {
+  moviesConnection(first: 10, after: "opaqueCursor", sort: [{ title: ASC }]) {
+    pageInfo {
+      endCursor
+      hasNextPage
+      hasPreviousPage
+      startCursor
+    }
+    edges {
+      cursor
+      node {
+        title
+      }
+    }
+  }
+}
+```
+Version 6:
+```graphql
+query {
+  movies {
+    connection(
+      first: 10
+      after: "opaqueCursor"
+      sort: [{ node: { title: ASC } }]
+    ) {
+      pageInfo {
+        hasPreviousPage
+        hasNextPage
+        startCursor
+        endCursor
+      }
+      edges {
+        cursor
+        node {
+          title
+        }
+      }
+    }
+  }
+}
+```
+
+Note how in version 6, the sort field `title` is now nested in the `node` field.
+
+*Nested field pagination and sorting*
+
+Version 5:
+```graphql
+query {
+  moviesConnection {
+    edges {
+      node {
+        title
+        actorsConnection(
+          first: 10
+          after: "opaqueCursor"
+          sort: [{ edge: { year: ASC } }, { node: { name: ASC } }]
+        ) {
+          edges {
+            cursor
+            node {
+              name
+            }
+            properties {
+              year
+            }
+          }
+          pageInfo {
+            hasNextPage
+            endCursor
+            hasPreviousPage
+            startCursor
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Version 6:
+
+```graphql
+query {
+  movies {
+    connection {
+      edges {
+        node {
+          title
+          actors {
+            connection(
+              sort: [
+                { edges: { properties: { year: ASC } } }
+                { edges: { node: { name: ASC } } }
+              ]
+            ) {
+              pageInfo {
+                endCursor
+                hasNextPage
+                hasPreviousPage
+                startCursor
+              }
+              edges {
+                cursor
+                node {
+                  name
+                }
+                properties {
+                  year
+                }
+              }
+            }
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+Note how in version 6, the sort field `year` is now nested in the `properties` field while the field `name` is nested in the `node` field, and they are both part of the `edges` field.
+
+=== Filtering 
+
+Filtering came with a lot of changes. 
+In version 5, we used to define new filters by appending the name of the filter to the name of the field, like: `title_CONTAINS` or `title_STARTS_WITH`. 
+In the past, that has meant that every new field added in a GraphQL type by the user is reflected in the creation of several input fields by the Library.  In version 6 property filters are generated to be shared across different fields of the same type.
+Here an examples of some of the new filters available:
```

Comment:
Again, maybe reducing the amount of backstory. Focusing a bit more in the new rather than the old